source/common/json/config_schemas.cc

```diff
@@ -832,6 +832,44 @@ const std::string Json::Schema::FAULT_HTTP_FILTER_SCHEMA(R"EOF(
   }
   )EOF");
 
+const std::string Json::Schema::IP_TAGGING_HTTP_FILTER_SCHEMA(R"EOF(
+  {
+    "$schema": "http://json-schema.org/schema#",
+    "type" : "object",
+    "properties" : {
+      "request_type" : {
+        "type" : "string",
+        "enum" : ["internal", "external", "both"]
+      },
+      "ip_tags" : {
+        "type" : "array",
+        "minItems" : 1,
+        "items" : {
+          "type" : "object",
+          "properties" : {
+            "ip_tag_name" : {
+              "type" : "string"
+            },
+            "ip_list" : {
+              "type" : "array",
+              "minItems" : 1,
+              "uniqueItems": true,
+              "items" : {
+                "type": "string",
+                "oneOf": [
+                  { "pattern" : "^([0-9]{1,3}\.){3}[0-9]{1,3}(\/([0-9]|[1-2][0-9]|3[0-2]))?$" },
```

Comment:
from the comment above: `In fact I think we need to change this to a regex match, because as far as I can tell there is no pre determined format for a CIDR range. And what this is is actually an array of cidr ranges, isn't it?`